index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio mobile version skeleton</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    
+    <!-- Header -->
+   <header class="header">
+      <div class="logo">
+        John Doe
+      </div>
+      <div class="menu-icon-container">
+        <i class="fa fa-bars" aria-hidden="true"></i>
+      </div>
+   </header>
+
+   <section class="headline-section">
+     
+     <div class="my-information">
+       <h1>Hey There.I’m Habeeb</h1>
+       <h4>I’am a Software Developer</h4>
+       <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      </div>
+      
+      <div class="social-media-links">
+        <a href="#"> <img src="images/icons/github.png" alt=""> </a>
+        <a href="#"> <img src="images/icons/linkedin.png" alt=""> </a>
```

Comment:
- [REQUIRED] Please, try to replace the `div` tag with a `ul` and putt all icons inside as child `li` elements. This is more semantic.

```html
<ul class="social-media">
  <li>
    <a href="#"> <img src="images/icons/github.png" alt=""> </a>
  </li>
  <li>
    <a href="#"> <img src="images/icons/linkedin.png" alt=""> </a>
  </li>
 <!-- rest of code -->
</ul>
```

Reply:
@OmarRamoun i have made all the required changes, thank you. should i resubmit

Reply:
yes @Efiamotu-1

Reply:
@Efiamotu-1. You forgot to push the code.

Reply:
i am very sorry, i have done that